ui/raidboss/data/06-ew/raid/p11n.txt

```diff
@@ -1,6 +1,149 @@
 ### P11N: Anabaseios: The Eleventh Circle
 
+# -ii 81B3 81B4 81B8 81C5 81C6 81D9 81DC 81E4 81E5 81F8 81F9 8790 8791 87CE 87CF
+
+# 81B3 Divisive Ruling
+# 81B4 Divisive Ruling
+# 81B8 Ripples of Gloom
+# 81C5 Katakrisis
+# 81C6 Katakrisis
+# 81D9 Ripples of Gloom
+# 81DC Styx
+# 81E4 Dike
+# 81E5 Dike
+# 81F8 Dismissal Ruling
+# 81F9 Dismissal Ruling
+# 8790 Katakrisis
+# 8791 Katakrisis
+# 87CE Upheld Ruling
+# 87CF Upheld Ruling
+
+
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+7.4 "--sync--" sync / 14:[^:]*:Themis:81E2:/ window 7.4,10
+12.1 "Eunomia" sync / 1[56]:[^:]*:Themis:81E2:/
+
+# Divisive Ruling Intro
+16.2 "--center--" sync / 1[56]:[^:]*:Themis:8235:/
+25.5 "Divisive Ruling (in)" sync / 1[56]:[^:]*:Themis:81B5:/
+28.2 "Ripples of Gloom" sync / 1[56]:[^:]*:Themis:(81B7|81B8):/
+37.8 "Divisive Ruling (out)" sync / 1[56]:[^:]*:Themis:81B5:/
+40.4 "Divine Ruination" sync / 1[56]:[^:]*:Themis:81B6:/
+50.1 "Divisive Ruling (random)" sync / 1[56]:[^:]*:Themis:81B5:/
+52.7 "Divine Ruination/Ripples of Gloom" sync / 1[56]:[^:]*:Themis:(81B6|81B7|81B8):/
+60.9 "Eunomia" sync / 1[56]:[^:]*:Themis:81E2:/
+
+# Dark and Light Intro
+67.0 "--center--" sync / 1[56]:[^:]*:Themis:8235:/ window 30,30
+73.1 "Dark and Light" sync / 1[56]:[^:]*:Themis:81C3:/
+80.2 "Sigils of Discord" sync / 1[56]:[^:]*:Themis:81C4:/ window 76.2,15
```

Comment:
I'm not sure I understand this `window`, or the other ones below in Upheld Ruling or Styx x5?

There's also a lot of `window 30,30` etc; is this really inconsistent, or just adding guide rails to make sure everything stays on track?

Reply:
Whoops, I knew I was going to miss something. I had to adjust the entire timeline four seconds later because the log I generated the base timeline from was early.

As for why there are so many safety syncs on an otherwise on-rails timeline, we've had situations before where pushes are discovered later. This is an attempt to get out ahead of that.